convex/queries.test.ts

```diff
@@ -53,6 +53,55 @@ test("withIndex with undefined", async () => {
   expect(things).toMatchObject([{}]);
 });
 
+describe("system index", () => {
+  const schema = defineSchema({
+    things: defineTable({
+      field: v.string(),
+    }).index("field", ["field"]),
+  });
+  let t: TestConvex<typeof schema>;
+  beforeEach(() => {
+    vi.useFakeTimers();
+    t = convexTest(schema);
+  });
+
+  afterEach(() => {
+    vi.useRealTimers();
+  });
```

Comment:
nit: We don't really need this? You can put the fake timers in the one test that needs it, and you probably don't need a custom schema?

Reply:
I personally prefer having lifecycle hooks for things that have clear set up + tear down. I missed that there was an existing schema I should've used.

Reply:
I wish vitest had a better setup for this time faking. When would you ever want to enable it in a test but then not disable it? This can be done via configuration, but it would be too magical for people to use this repo as an example.